rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/common/event/RMNodeSourceEvent.java

```diff
@@ -67,7 +66,7 @@
     private String nodeSourceName = null;
 
     /** description of the source concerned by the event. */
-    private BigString nodeSourceDescription = null;
+    private String nodeSourceDescription = null;
```

Comment:
Have you tried to create a node source with a really long description just in case (after having recompiled and removed database)?